app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    memory = {}
+
+    def inner(*args, **kwargs) -> int:
+        array = tuple([*args])
```

Comment:
The conversion to a tuple using `tuple([*args])` is unnecessary. You can directly use `args` since it is already a tuple. Consider using `args` directly to simplify the code.